plugin.video.youtube/resources/lib/youtube/client/login_client.py

```diff
@@ -19,83 +35,57 @@ class LoginClient(object):
         'youtube-for-kodi-quota': {
             'token-allowed': False,
             'system': 'All',
-            'key': 'AIzaSyA7v1QOHz8Q4my5J8uGSpr0zRrntRjnMmk',
-            'id': '597640352045-7um2gr1v5rgobm5bf07ebesm3er48286.apps.googleusercontent.com',
-            'secret': 'VmyQ12KkJ_N3yegu4Y-VGCXd'
+            'key': '%s' % api_key,
```

Comment:
No problem, just wanted to make sure.

Thanks for all your advice and help!

Jeff


> On May 23, 2016, at 2:47 PM, Philipp Temminghoff <notifications@github.com> wrote:
> 
> In plugin.video.youtube/resources/lib/youtube/client/login_client.py <https://github.com/xbmc/repo-plugins/pull/480#discussion_r64264574>:
> 
> > @@ -19,83 +35,57 @@ class LoginClient(object):
> >          'youtube-for-kodi-quota': {
> >              'token-allowed': False,
> >              'system': 'All',
> > -            'key': 'AIzaSyA7v1QOHz8Q4my5J8uGSpr0zRrntRjnMmk',
> > -            'id': '597640352045-7um2gr1v5rgobm5bf07ebesm3er48286.apps.googleusercontent.com',
> > -            'secret': 'VmyQ12KkJ_N3yegu4Y-VGCXd'
> > +            'key': '%s' % api_key,
> you dont have to do it at all, just a remark. ;)
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/xbmc/repo-plugins/pull/480/files/5ab7db1a24d3b6ff8e2f8a4d594407523dcf890a#r64264574>